public/js/admin.js

```diff
@@ -311,16 +308,13 @@ var activateUser = function(id) {
             success: function(data) {
                 displayAccountsTable();
                 const msg = ' has been activated';
-                const colour = colours.green;
-                const icon = '<i class="material-icons">check</i>&nbsp&nbsp&nbsp';
-                dropSnack(colour, icon + id + ' account' + msg);
+                successSnackbar(id + ' account' + msg);
```

Comment:
Id 's account

Reply:
why do we need variable `msg` in this case? why cant we just have `successSnackbar(id + ''s account has been activated');`

Reply:
Not sure, that was there from before. I thought someone wanted to add some extra functionality there later. Ill mend it. 👍